movement/watch_faces/complication/countdown_face.c

```diff
@@ -93,15 +119,24 @@ static void draw(countdown_state_t *state, uint8_t subsecond) {
     watch_display_string(buf, 0);
 }
 
+static void pause(countdown_state_t *state) {
+    state->mode = cd_paused;
+    movement_cancel_background_task();
+    watch_clear_indicator(WATCH_INDICATOR_BELL);
+}
+
 static void reset(countdown_state_t *state) {
-    state->mode = cd_waiting;
+    state->mode = cd_reset;
     movement_cancel_background_task();
     watch_clear_indicator(WATCH_INDICATOR_BELL);
+    load_countdown(state);
 }
 
 static void ring(countdown_state_t *state) {
     movement_play_alarm();
-    reset(state);
+    movement_cancel_background_task();
+    watch_clear_indicator(WATCH_INDICATOR_BELL);
+    state->mode = cd_paused;
```

Comment:
Can you also restore the saved timer value here? This is what other casio timers usually do too :)